models/customer-group/src/generator.ts

```diff
@@ -0,0 +1,24 @@
+import { ClientLogging } from '@commercetools-test-data/commons';
+import { sequence, fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TCustomerGroup } from './types';
+
+// https://docs.commercetools.com/api/projects/customerGroups#customergroup
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCustomerGroup>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    version: sequence(),
+    key: fake((f) => f.lorem.slug(2)),
+    name: fake((f) => f.lorem.words(2)),
+    custom: [],
```

Comment:
nit: I don't think `CustomFields` is an array, perhaps `null` here might work?

Reply:
Wow you are 100% correct, I'll update all of these to null